aikau/src/main/resources/alfresco/dashlets/Dashlet.js

```diff
@@ -18,263 +18,368 @@
-             */
-            i18nRequirements: [{i18nFile: "./i18n/Dashlet.properties"}],
+         /**
+          * An array of the i18n files to use with this widget.
+          *
+          * @instance
+          * @type {object[]}
+          * @default [{i18nFile: "./i18n/Dashlet.properties"}]
+          */
+         i18nRequirements: [{
+            i18nFile: "./i18n/Dashlet.properties"
+         }],
 
-            /**
-             * An array of the CSS files to use with this widget.
-             *
-             * @instance cssRequirements {Array}
-             * @type {object[]}
-             * @default [{cssFile:"./css/Dashlet.css"}]
-             */
-            cssRequirements: [{cssFile:"./css/Dashlet.css"}],
+         /**
+          * An array of the CSS files to use with this widget.
+          *
+          * @instance cssRequirements {Array}
+          * @type {object[]}
+          * @default [{cssFile:"./css/Dashlet.css"}]
+          */
+         cssRequirements: [{
+            cssFile: "./css/Dashlet.css"
+         }],
 
-            /**
-             * We use the DashletResizer to handle the resizing.
-             *
-             * @instance
-             * @type {array}
-             */
-            nonAmdDependencies: ["/js/share.js"],
+         /**
+          * The HTML template to use for the widget.
+          * @instance
+          * @type {String}
+          */
+         templateString: template,
 
-            /**
-             * The HTML template to use for the widget.
-             * @instance
-             * @type {String}
-             */
-            templateString: template,
+         /**
+          * Config that makes it possible to dynamically update the title.
+          * Should be supplied in the format of
+          * {
+          *    titleConfig: {
+          *       topic: "SOME_TOPIC",
+          *       attribute: "path.to.topic.payload.attribute"
+          *    }
+          * }
+          *
+          * If there is no attribute present the entire payload value will be used as the title.
+          *
+          * @instance
+          * @type {object}
+          */
+         titleConfig: null,
 
-            /**
-             * Config that makes it possible to dynamically update the title.
-             * Should be supplied in the format of
-             * {
-             *    titleConfig: {
-             *       topic: "SOME_TOPIC",
-             *       attribute: "path.to.topic.payload.attribute"
-             *    }
-             * }
-             *
-             * If there is no attribute present the entire payload value will be used as the title.
-             *
-             * @instance
-             * @type {object}
-             */
-            titleConfig: null,
+         /**
+          * The container that holds the title.
+          * Will be populated by dojo.
+          *
+          * @instance
+          * @type {HTMLElement}
+          */
+         titleNode: null,
```

Comment:
I'm not sure we need to define all of these nodes that are declared in the HTML template, we don't do it anywhere else and it doesn't add a huge amount of value... and it makes the source file that much longer. I appreciate you're just leaving what was there before, but it might be worth removing them.